todoapp/app/build.gradle

```diff
@@ -80,7 +80,7 @@ dependencies {
 
     // Android Testing Support Library's runner and rules
     androidTestCompile "com.android.support.test:runner:$rootProject.ext.runnerVersion"
-    androidTestCompile "com.android.support.test:rules:$rootProject.ext.runnerVersion"
+    androidTestCompile "com.android.support.test:rules:$rootProject.ext.rulesVersion"
```

Comment:
As both of `rulesVersion` and `runnerVersion` exist in `android-architecture/todoapp/build.gradle` and have the same  `0.5` value (no surprising, they're always have the same) better solution would be to _join them_ into one like `androidTestLibVersion` instead of keeping them both. 

Just a suggestion ;-)